runtime/interpreter/value.go

```diff
@@ -12772,7 +12835,12 @@ func ConvertFix64(value Value) Fix64Value {
 		if value > Fix64MaxValue {
 			panic(OverflowError{})
 		}
-		return Fix64Value(value)
+		return NewFix64Value(
+			memoryGauge,
+			func() int64 {
+				return int64(value)
+			},
+		)
 
 	case BigNumberValue:
 		v := value.ToBigInt()
```

Comment:
Needs to be metered before this allocation occurs. See e.g. `ConvertInt64`

Reply:
ah yes, I had the entire function as the converter, but then refactored to make it a bit cleaner, and have missed these two in the process. Fixed them in: b669fa35e80b4803f1ad35551c79062d4b4a8d0b